apple/RNCWebViewManager.m

```diff
@@ -286,4 +313,21 @@ - (BOOL)          webView:(RNCWebView *)webView
   }
 }
 
+RCT_EXPORT_METHOD(releaseWebView:(nonnull NSString *)webViewKey)
+{
+  [self.bridge.uiManager addUIBlock:^(__unused RCTUIManager *uiManager, __unused NSDictionary<NSNumber *, RNCWebView *> *viewRegistry) {
```

Comment:
> Is https://github.com/facebook/react-native/blob/d9f7c4c189cf6d2ccd4b6641599b5e46898f7d0d/packages/react-native/React/Fabric/Mounting/RCTComponentViewRegistry.h the view registry you’re referring to?

Well, I believe I was specifically referring to the `viewRegistry` dictionary that's passed into this function via `uiManager addUIBlock` and can return a native view via `viewRegistry[reactTag]` (example [here](https://reactnative.dev/docs/native-components-ios#handling-multiple-native-views))... but I think those are related?

> I haven’t yet found an equivalent of the releaseWebView API added in this PR for releasing the native view

I thought "releasing" was specific to the `WKWebView`, not the outer `RNCWebView`? I think I had only been suggesting the `viewRegistry` as an alternative to `sharedRNCWebViewDictionary`, not `sharedWKWebViewDictionary`

> a view that would temporarily be the parent of the native WebView

Interesting. That doesn't cause side-effects?


Well, _anyhow_, curiosities aside, this suggestion may not be so relevant now that you've started using `superview` instead of `sharedRNCWebViewDictionary`, and also relying on stuff like the `viewRegistry` may not be a good idea with the new architecture (though @Titozzz is the expert there, not me).